vlsi/example-tech.yml

```diff
@@ -0,0 +1,10 @@
+# Technology Setup
```

Comment:
Are we sure this needs to be specified here? Don't the hammer-tech default.ymls supply these?

Reply:
I'm going to defer this question to @colinschmidt @harrisonliew

Reply:
Generally, the plugin will supply `vlsi.core.node` only. However, the rest of the keys before this imo should be explicit (otherwise, how do you know which tech you're even using?). `technology.<tech_name>.install_dir` is also end environment-specific, and in the case of ASAP7, comes as a tarball instead of a directory.

Reply:
I agree that specifying the tech you're using, the path to the plugin and the path to the PDK is important. However, do we really need `vlsi.core.technology_path_meta` and `vlsi.core.node` to be manually specified rather than just using default values from the tech plugin? 
Default values are a good thing!

Reply:
I don't think the `vlsi.core.node` needs to be specified here since it should be specified in the tech plugin.

Reply:
Re the path_meta variable. Yes we need that so that hammer knows how to handle conflicting assignments of the tech_path key. With append it will add the path in this file to a list of paths to look for tech files.